prosemirror-jdita/src/plugin.ts

```diff
@@ -5,28 +5,69 @@ import { toggleMark, newLine, hasMark, insertNode, insertImage, InputContainer }
 import { Command } from "prosemirror-commands";
 import { redo, undo } from "prosemirror-history";
 
+/**
```

Comment:
The part `Retrieve the DOM element with ID "editor" and assign it to a constant targetNode` is completely redundant as the intention of the code is clear.

Reply:
Fixed with 641a6a2.